libs/search-widget/src/core/old-stores/main.store.ts

```diff
@@ -95,6 +96,23 @@ export const setDisplayedResource = (resource: DisplayedResource) => {
   nucliaStore().displayedResource.next(resource);
 };
 
+export const addLabelFilter = (label: Classification) => {
+  const filter = getFilterFromLabel(label);
+  const currentFilters = nucliaStore().filters.value;
+  nucliaStore().filters.next(currentFilters.concat([filter]));
```

Comment:
are we sure it is not possible the label was already in the list?

Reply:
good point